crates/shared/src/db_order_conversions.rs

```diff
@@ -115,19 +110,30 @@ pub fn order_kind_from(kind: DbOrderKind) -> OrderKind {
     }
 }
 
-pub fn order_class_into(class: OrderClass) -> DbOrderClass {
+pub fn order_class_into(class: &OrderClass) -> DbOrderClass {
     match class {
         OrderClass::Market => DbOrderClass::Market,
         OrderClass::Liquidity => DbOrderClass::Liquidity,
-        OrderClass::Limit => DbOrderClass::Limit,
+        OrderClass::Limit(_) => DbOrderClass::Limit,
     }
 }
 
-pub fn order_class_from(class: DbOrderClass) -> OrderClass {
-    match class {
+pub fn order_class_from(order: &FullOrderDb) -> OrderClass {
+    match order.class {
         DbOrderClass::Market => OrderClass::Market,
         DbOrderClass::Liquidity => OrderClass::Liquidity,
-        DbOrderClass::Limit => OrderClass::Limit,
+        DbOrderClass::Limit => OrderClass::Limit(LimitOrderClass {
+            surplus_fee: big_decimal_to_u256(
+                order
+                    .surplus_fee
+                    .as_ref()
+                    .expect("limit orders must have surplus fee set"),
+            )
+            .unwrap(),
+            surplus_fee_timestamp: order
+                .surplus_fee_timestamp
+                .expect("limit orders must have surplus fee timestamp set"),
```

Comment:
Not sure if we did allow limit orders with null fee or timestamp before this PR. In that case we might need a DB migration to patch up those orders.

Reply:
I'm not concerned about the non-limit order because they wouldn't be affected by the `.expect()`. Only limit orders with null values would panic here. But I also think those are probably not a big problem.

Reply:
Looks like we don't have any such orders in the DB so we would be alright if we don't create new ones in the mean time. 👌